File: CS/DXSample/AbsoluteVisible.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Windows;

namespace DXSample
{
    public class AbsoluteVisible
    {
        public static readonly DependencyProperty IsEnableProperty = DependencyProperty.RegisterAttached("IsEnable", typeof(bool), typeof(AbsoluteVisible), new UIPropertyMetadata(null));

        public static bool GetIsEnable(DependencyObject target)
        {
            return (bool)target.GetValue(IsEnableProperty);
        }

        public static void SetIsEnable(DependencyObject target, bool value)
        {
            target.SetValue(IsEnableProperty, value);
        }     
    }
}

File: VB/DXSample/AbsoluteVisible.vb
Imports System
Imports System.Collections.Generic
Imports System.Linq
Imports System.Text
Imports System.Windows

Namespace DXSample
    Public Class AbsoluteVisible
        Public Shared ReadOnly IsEnableProperty As DependencyProperty = DependencyProperty.RegisterAttached("IsEnable", GetType(Boolean), GetType(AbsoluteVisible), New UIPropertyMetadata(Nothing))

        Public Shared Function GetIsEnable(ByVal target As DependencyObject) As Boolean
            Return DirectCast(target.GetValue(IsEnableProperty), Boolean)
        End Function

        Public Shared Sub SetIsEnable(ByVal target As DependencyObject, ByVal value As Boolean)
            target.SetValue(IsEnableProperty, value)
        End Sub
    End Class
End Namespace
